release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -735,9 +735,9 @@ When configuring the `docker/config.json` file to allow pods to pull images from
 Node-related metrics and alerts have been enhanced to give you an earlier indication of when the stability of a node is compromised.
 
 [id="ocp-4-9-poison-pill-operator"]
-==== Enhanced remediation with the Poison Pill Operator 
+==== Enhanced remediation with the Poison Pill Operator
 
-You can use the Poison Pill Operator regardless of the cluster installation type, such as Installer Provisioned Infrastructure (IPI) or User Provisioned Infrastructure (UPI). 
+You can use the Poison Pill Operator regardless of the cluster installation type, such as Installer Provisioned Infrastructure (IPI) or User Provisioned Infrastructure (UPI).
```

Comment:
Is this statement needed? Doesn't this apply to most/all operators? 

cc: @davemulford

Reply:
+1 Eric. I don't think this sentence is needed.